docs/source/tutorials/day_of_the_week.qmd

```diff
@@ -0,0 +1,309 @@
+
+For this example, the effect will be passed as a truncated normal distribution with a mean of 1.0 and a standard deviation of 0.5. The distribution will be truncated between 0.1 and 10.0[^note-other-examples]:
+
+[^note-other-examples]: A similar weekday effect is implemented in its own module, with example code [here](periodic_effects.html).
+
+```{python}
+# | label: weekly-effect
+# Initializing the RV.
+from pyrenew.metaclass import DistributionalRV
+
+dayofweek_effect = DistributionalRV(
+    "dayofweek_effect",
+    dist.TruncatedNormal(loc=1.0, scale=0.5, low=0.1, high=10.0).expand([7]),
+)
+```
+
+Now, by re-defining the latent hospital admissions random variable with the day-of-the-week effect, we can build a model that includes this effect:
+
+```{python}
+# | label: latent-hosp-weekday
+latent_hosp_wday_effect = latent.HospitalAdmissions(
+    infection_to_admission_interval_rv=inf_hosp_int,
+    infect_hosp_rate_rv=hosp_rate,
+    day_of_week_effect_rv=dayofweek_effect,
+    # As the dataset starts on October 30th, 2023 (Monday)
+    # there's no need to offset the day-of-the-week effect
+    obs_data_first_day_of_the_week=0,
+)
+
+# New model with day-of-the-week effect
+hosp_model_dow = model.HospitalAdmissionsModel(
+    latent_infections_rv=latent_inf,
+    latent_hosp_admissions_rv=latent_hosp_wday_effect,
+    I0_rv=I0,
+    gen_int_rv=gen_int,
+    Rt_process_rv=rtproc,
+    hosp_admission_obs_process_rv=obs,
+)
+```
+
+Running the model:
+
+```{python}
+# | label: model-2-run-weekday
+# Model with weekday effect
+hosp_model_dow.run(
+    num_samples=2000,
+    num_warmup=2000,
+    data_observed_hosp_admissions=daily_hosp_admits,
+    rng_key=jax.random.PRNGKey(54),
+    mcmc_args=dict(progress_bar=False),
+)
+```
+
+As a result, we can see the posterior distribution of our novel day-of-the-week effect:
+
+```{python}
+# | label: fig-output-day-of-week
+# | fig-cap: Day of the week effect
+out = hosp_model_dow.plot_posterior(
+    var="dayofweek_effect", ylab="Day of the Week Effect", samples=500
+)
+```
+
+The new model with the day-of-the-week effect can be compared to the previous model without the effect. Let's start by reproducing the figure without the day-of-the-week effect, and then plotting the new model with the effect:
```

Comment:
> Let's start by reproducing the figure without the day-of-the-week effect, and then plotting the new model with the effect:

Seems to suggest we will be doing more after plotting the figures, but that's all we do.

Reply:
Good point